app/assets/javascripts/discourse/app/components/post-text-selection.gjs

```diff
@@ -257,29 +230,63 @@ export default class PostTextSelection extends Component {
   }
 
   @bind
-  onSelectionChanged() {
-    if (this.isSelecting) {
-      return;
-    }
+  contextmenu() {
+    this.handleSelection();
+    this.enablePointerEventsOnMenu();
+  }
 
-    const { isIOS, isWinphone, isAndroid } = this.capabilities;
-    const wait = isIOS || isWinphone || isAndroid ? INPUT_DELAY : 25;
-    this.selectionChangeHandler = discourseDebounce(
```

Comment:
now that we rely on pointerup and contextmenu and not selectionchange, we don't need debouncing anymore.